README.md

```diff
@@ -467,6 +467,9 @@ values stored in `argv._`).  Set `desc` to `false` to create a hidden command.
 Hidden commands don't show up in the help output and aren't available for
 completion.
 
+Note that commands will **not** automatically inherit the configuration of their
```

Comment:
Let's move this documentation below

> Optionally, you can provide a `builder` object to give hints about the options that your command accepts:

And let's make it sound a bit more positive, pointing folks towards the `globals` option 👍 

> By default only [global](#global) commands are inherited by the yargs instance passed to the builder.